unison-src/transcripts-using-base/doc.output.md

```diff
@@ -114,7 +114,7 @@ and the rendered output using `display`:
       section elements.
       
       Text can be **bold**, __italicized__, ~~strikethrough~~,
-      or ''monospaced''.
+      or ''monospaced'' (or ''monospaced'').
```

Comment:
This is no big deal but also kind of weird. Do we actually want two syntaxes, and if so is this the one that we want shown after a round trip? Note that the rendered markdown below shows it as `` `monospaced` ``.